srcpkgs/dotnet-sdk-bin/template

```diff
@@ -0,0 +1,56 @@
+# Template file for 'dotnet-sdk-bin'
+pkgname=dotnet-sdk-bin
+version=5.0.101
+revision=1
+archs="x86_64* aarch64"
+create_wrksrc=yes
+short_desc="SDK for making dotnet CORE apps"
+maintainer="KawaiiAmber <japaneselearning101@gmail.com>"
+license="MIT"
+homepage="https://dotnet.microsoft.com"
+
+_target=${DESTDIR}/opt/dotnet
```

Comment:
Please put this line together with `_prof`, down below.

Reply:
Done